src/avfoundation.cs

```diff
@@ -3233,6 +3243,11 @@ interface AVSampleBufferAudioRenderer : AVQueuedSampleBufferRendering
 		[NullAllowed, Export ("error")]
 		NSError Error { get; }
 
+#if XAMCORE_4_0
+		[NoWatch, NoTV, NoiOS]
```

Comment:
our mistake ? or a change in the headers ?

Reply:
Change in headers

Reply:
Better add an empty stub in `AVCompat.cs` for the non-Mac platforms. This avoid the extraneous generated code and any reference to the selector (in case it gets rejected in the future).